atlasdb-api/src/main/java/com/palantir/atlasdb/config/LeaderConfig.java

```diff
@@ -47,15 +49,13 @@ public File acceptorLogDir() {
     public abstract String localServer();
 
     @Size(min = 1)
-    public abstract Set<String> leaders();
+    public abstract List<String> leaders();
```

Comment:
Talked to Joe again. Since most people seem to prefer it to be a `Set`, I'll change it back and put in a docs change so that it no longer says we pick the first in the list. We can't check for duplicates entries anymore though.